src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -0,0 +1,221 @@
+            // Handle non-success status codes
+            if (response.statusCode !== 200) {
+                file.close();
+                fs.unlink(dest, (unlinkErr) => {
+                    if (unlinkErr) {
+                        console.warn(`Warning: Failed to clean up temporary file '${dest}' after HTTP error: ${unlinkErr.message}`);
+                    }
+                    reject(new Error(`Failed to get '${fileUrl}' (${response.statusCode})`));
+                });
+                return;
+            }            // Get the total file size from the response headers
+            const totalBytes = parseInt(response.headers && response.headers['content-length'] || '', 10);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)